NCS.DSS.Address/PostAddressHttpTrigger/Function/PostAddressHttpTrigger.cs

```diff
@@ -61,72 +59,69 @@ public class PostAddressHttpTrigger
         {
-            log.LogInformation($"Started Executing Address POST Request for CustomerId [{customerId}]");
+            _logger.LogInformation($"Started Executing Address POST Request for CustomerId [{customerId}]");
 
             var correlationId = _httpRequestHelper.GetDssCorrelationId(req);
             if (string.IsNullOrEmpty(correlationId))
-                log.LogWarning("Unable to locate 'DssCorrelationId; in request header");
+                _logger.LogWarning("Unable to locate 'DssCorrelationId; in request header");
 
             if (!Guid.TryParse(correlationId, out var correlationGuid))
             {
                 correlationGuid = Guid.NewGuid();
-                log.LogWarning($"Unable to Parse 'DssCorrelationId' to a Guid. New Guid Generated");
+                _logger.LogWarning($"Unable to Parse 'DssCorrelationId' to a Guid. New Guid Generated");
             }
 
-            log.LogInformation($" 'DssCorrelationId' is [{correlationGuid}]");
+            _logger.LogInformation($" 'DssCorrelationId' is [{correlationGuid}]");
 
             var touchpointId = _httpRequestHelper.GetDssTouchpointId(req);
             if (string.IsNullOrEmpty(touchpointId))
-                return ReturnBadRequest( log,"Unable to locate 'APIM-TouchpointId' in request header.");
+                return ReturnBadRequest("Unable to locate 'APIM-TouchpointId' in request header.");
 
             var ApimURL = _httpRequestHelper.GetDssApimUrl(req);
             if (string.IsNullOrEmpty(ApimURL))
-                return ReturnBadRequest( log,"Unable to locate 'apimurl' in request header.");
+                return ReturnBadRequest("Unable to locate 'apimurl' in request header.");
 
             var subContractorId = _httpRequestHelper.GetDssSubcontractorId(req);
             if (string.IsNullOrEmpty(subContractorId))
-                log.LogInformation( "Unable to locate 'SubContractorId' in request header. Continuing POST Process");
+                _logger.LogInformation("Unable to locate 'SubContractorId' in request header. Continuing POST Process");
 
-            log.LogInformation("Post Address C# HTTP trigger function  processed a request. By Touchpoint " + touchpointId);
+            _logger.LogInformation("Post Address C# HTTP trigger function  processed a request. By Touchpoint " + touchpointId);
 
             if (!Guid.TryParse(customerId, out var customerGuid))
-                return ReturnBadRequest( log,"Unable to Parse customerId to Guid",customerGuid);
+                return ReturnBadRequest("Unable to Parse customerId to Guid", customerGuid);
 
             Models.Address addressRequest;
 
             try
             {
                 addressRequest = await _httpRequestHelper.GetResourceFromRequest<Models.Address>(req);
             }
-            catch (JsonException ex)
+            catch (Newtonsoft.Json.JsonException ex)
```

Comment:
You will have to capture _**Exception**_ instead of JsonException as the _httpRequestHelper throws Newtonsoft.Json.JsonException and we are avoiding to use it in repos.